packages/clay-css/src/images/icons/list-ol-rtl.svg

```diff
@@ -0,0 +1,11 @@
+<!--
+* Clay 3.12.0
+*
+* SPDX-FileCopyrightText: © 2020 Liferay, Inc. <https://liferay.com>
+* SPDX-FileCopyrightText: © 2020 Contributors to the project Clay <https://github.com/liferay/clay/graphs/contributors>
+*
+* SPDX-License-Identifier: BSD-3-Clause
+-->
```

Comment:
Oh, I figured we weren't going to add that to the icon files, I realized that some that I merged yesterday didn't contain the copyright 😅.

Reply:
@matuzalemsteles It's ok I added them at https://github.com/liferay/clay/pull/3406. I sent the pulls before I merged the license stuff.

Reply:
Oh great, thanks @pat270